src/ngx_http_lua_module.c

```diff
@@ -1071,6 +1072,12 @@ ngx_http_lua_init_main_conf(ngx_conf_t *cf, void *conf)
         lmcf->worker_thread_vm_pool_size = 100;
     }
 
+    if (ngx_http_lua_init_builtin_headers_out(cf, lmcf) != NGX_OK) {
+        ngx_conf_log_error(NGX_LOG_EMERG, cf, 0, "init header out error");
+
+        return NGX_CONF_ERROR;
+    }
+
```

Comment:
maybe it's worth a debug log for the hash size after init?

Reply:
ok, I agree